lib/cbus_elixir_web/templates/meeting/form.html.eex

```diff
@@ -0,0 +1,35 @@
+<%= form_for @changeset, @action, fn f -> %>
+  <%= if @changeset.action do %>
+    <div class="alert alert-danger">
+      <p>Oops, something went wrong! Please check the errors below.</p>
+    </div>
+  <% end %>
+
+  <div class="row">
+    <div class="column" style="flex-direction: row;">
+      <div class="form-group date-picker">
+        <%= label f, :date, class: "control-label" %>
+        <%= date_select f, :date, class: "form-control" %>
```

Comment:
Note that date_select isn't compatible with utc_datetime. I figure we probably don't need to store the time anyways, and it looks like we aren't in the seeds, so I figured the migration was the best thing to do. Let me know if I'm wrong.